libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -1,3 +1,8 @@
+:root {
+  --pep-background-prompt: #ffffff;
+  --pep-background-progress: #e9e9e9;
+}
```

Comment:
A final nit - these values are specific to PEP. `:root` would never need them, so why not add the custom properties inside `.appPrompt`?

Reply:
@overmyheadandbody Good point! Made the change 👍